geomesa-cassandra/geomesa-cassandra-datastore/src/main/scala/org/locationtech/geomesa/cassandra/data/CassandraFeatureStore.scala

```diff
@@ -65,6 +66,12 @@ class CassandraFeatureStore(entry: ContentEntry) extends ContentFeatureStore(ent
   }
 
 
+  override def getInfo: ResourceInfo = {
```

Comment:
Sine we have this four places, I wonder if it makes sense to extract a tiny function to do this keyword enrichment?